### PR TITLE
docs: ibc relaying and chain upgrades

### PR DESCRIPTION
**Pull request by conorsch**
First pass at making sure the necessary steps for updating relayers across upgrade boundaries are well documented.

### CODE REVIEW

File: pages/node/pd/indexing-events.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ planned upgrade. The process then becomes:
 1. Restore node state from backup.
 2. Ensure you're using the appropriate `pd` and `cometbft` versions for the associated state.
 3. Run `pd migrate --ready-to-start` to permit `pd` to start up.
-4. Run CometBFT with extra options: `--p2p.pex=false --p2p.seeds='' --moniker archive-node-1`
+4. Run CometBFT with extra options: `--p2p.pex=false --p2p.seeds='' --p2p.persistent_peers='' --p2p.laddr="tcp://127.0.0.1:26656" --moniker archive-node-1`
```
Comment (conorsch) [inline pages/node/pd/indexing-events.md:31]: These options are based on feedback we received from TM/CMT folks on best practices. Trying to be specific, but there's actually a lot one needs to do to make sure a node is not talking to peers, like nuking the address book. Even if the node does peer and disagrees about block contents, it'll still serve its RPC, which means its still suitable for as an archive node, at least for relaying.

Comment (conorsch) [reply pages/node/pd/indexing-events.md]: Added a step to move aside the addrbook, after conferring with some relayer operators about best practices.

**Comment by conorsch** [comment]
Updated with latest steps. Ready for review!

**Review by aubrika — APPROVED** [review]
LGTM, great updates 🙏🏻

**Pull request merged**